cosmo/tests/utils.py

```diff
@@ -40,15 +40,22 @@ def __init__(self, **kwargs):
         self.text = kwargs['text']
 
     @staticmethod
-    def patchTool(mocker, returnData={'status_code': 200, 'text': '{"data": {"vrf_list": [], "device_list": []}}'}):
-        postMock = mocker.patch('requests.post', return_value=RequestResponseMock(**returnData))
-        return postMock
+    def patchTool(mocker, graphqlData=None, versionData=None):
+
+        if graphqlData is None:
+            graphqlData = {'status_code': 200, 'text': json.dumps({"data": {"vrf_list": [], "device_list": []}})}
+
+        if versionData is None:
+            versionData = {'status_code': 200, 'text': json.dumps({"netbox-version": "wc_3.7.5-0.7.0"})}
+
+        postMock1 = mocker.patch('requests.get', return_value=RequestResponseMock(**versionData))
+        postMock2 = mocker.patch('requests.post', return_value=RequestResponseMock(**graphqlData))
```

Comment:
smort :sunglasses: I like the get / post patching. It's elegant :+1: thank you!  :balloon: :rainbow: :sun_with_face: